app/lib/server/startup/settings.js

```diff
@@ -998,6 +998,34 @@ settings.addGroup('Message', function() {
 			public: true,
 			i18nDescription: 'Message_Attachments_GroupAttachDescription',
 		});
+
+		this.add('Message_Attachments_Thumbnails_Enabled', true, {
+			type: 'boolean',
+			public: true,
+			i18nDescription: 'Message_Attachments_Thumbnails_EnabledDesc',
+		});
+
+		this.add('Message_Attachments_Thumbnails_Width', 200, {
```

Comment:
according to frontend typings (https://github.com/RocketChat/Rocket.Chat/blob/3b003da3dc403764b293aa02709af23c7256623b/client/components/Message/Attachments/context/AttachmentContext.tsx#L17), image attachments have 480px, so I think we should at least have that as default:

```suggestion
		this.add('Message_Attachments_Thumbnails_Width', 480, {
```